homework-2/models/index.ts

```diff
@@ -21,8 +23,15 @@ const sequelize  = new Sequelize(
 );
 
 const User = userFactory(sequelize);
+const Group = groupFactory(sequelize);
+const UsersGroups = usersGroupsFactory(sequelize);
 
-export const db: IDB = {
+User.belongsToMany(Group, { through: UsersGroups });
+Group.belongsToMany(User, { through: UsersGroups });
+
+export const db: DB = {
```

Comment:
I cannot see a reason to use db object

Reply:
It groups all ways to access DB, using this object developer can understand that some part of the code works with DB, or during developing you can easily start to interact with DB and find any object that related to it.